### PR TITLE
pcsx2: manually cast function pointer to void*

### PR DESCRIPTION
**Pull request by gregory38**
Templace is nicer but give a hard time to compiler.

New version compile in both gcc&clang without hack

=> #1488

### CODE REVIEW

**Comment by turtleli** [comment]
Compiles fine with clang 3.8.1. Too bad VS2013 doesn't like it.

**Comment by gregory38** [comment]
Yeah. Maybe I need to add an (uptr) cast too :(

**Comment by gregory38** [comment]
Sigh... Uptr won't do it. Strange.

File: pcsx2/x86/microVU_Branch.inl [hunk 1]
```diff
@@ -192,8 +192,8 @@ void normJumpCompile(mV, microFlagCycles& mFC, bool isEvilJump) {
 		xJMP(mVU.exitFunct);
 	}
 	
-	if (!mVU.index) xFastCall(mVUcompileJIT<0>, gprT2, gprT3); //(u32 startPC, uptr pState)
-	else			xFastCall(mVUcompileJIT<1>, gprT2, gprT3);
+	if (!mVU.index) xFastCall((void*)(uptr)mVUcompileJIT<0>, gprT2, gprT3); //(u32 startPC, uptr pState)
+	else			xFastCall((void*)(uptr)mVUcompileJIT<1>, gprT2, gprT3);
```
Comment (turtleli) [inline pcsx2/x86/microVU_Branch.inl]: `(void*)(void(*)())mVU...` works.

Comment (gregory38) [inline pcsx2/x86/microVU_Branch.inl]: Thanks you

**Comment by gregory38** [comment]
Ok. Compilers are happy, let's merge it.

**Comment by danilaml** [comment]
Btw, are there any plans to drop VS2013 in the near future? VS2015 is all-around better.

**Comment by gregory38** [comment]
When ms release a new version of visual studio. 2013 supports most of the useful C++11 construct. It isn't blocking as previous version.

**Pull request merged**